docs-src/docs/setup/install.md

```diff
@@ -10,12 +10,14 @@ Before you can install and configure Layer0, you must obtain the following:
 ## Part 1: Download and extract Layer0
 
 1. In the [Downloads section of the home page](/index.html#download), select the appropriate installation file for your operating system. Extract the zip file to a directory on your computer.
-2. Add both the directory that contains the **l0** application, as well as the entire l0-setup directory, to your system path. The l0-setup directory contains files that are necessary for the **l0-setup** application to work properly, so you must add the entire directory to your path.<br />For more information about adding directories to your system path, see the following resources:
+2. (Optional) Place the **l0** and **l0-setup** binaries into your system path. 
+For more information about adding directories to your system path, see the following resources:
 	* (Windows): [How to Edit Your System PATH for Easy Command Line Access in Windows](http://www.howtogeek.com/118594/how-to-edit-your-system-path-for-easy-command-line-access/)
 	* (Linux/macOS): [Adding a Directory to the Path](http://www.troubleshooters.com/linux/prepostpath.htm)
 
 ## Part 2: Create an Access Key
-This step will create an Identity & Access Management (IAM) access key from your AWS account. You will use the credentials created in this section when installing, updating, or removing Layer0 resources.
+This step will create an Identity & Access Management (IAM) access key from your AWS account. 
```

Comment:
[nit typo] from = for. Possibly clarify in this sentence that the IAM key is specifically for Layer0.